### PR TITLE
Settingup contact form for portfolio

### PR DESCRIPTION
**Pull request by samuelmoiwa**
## Solution description

**CSS:** style file is designed to match the figma design file.
**HTML:**  HTML contents are added, to match the one from the figma design.
**Formspree:** form submission configure with formspace
Linters files: linters files configured

## Address affected and ensured
* HTML file
* CSS file
* Linters Config
---

### CODE REVIEW

**Review by tugsem — CHANGES_REQUESTED** [review]
Hi @samuelmoiwa , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!
Alt Text
## Highlights ⭐ 
✔️Repo is professional
✔️ Linters set up correctly and passing
✔️ Good job implementing styling

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -0,0 +1,505 @@
...
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+}
+
+.Get-In-Touch-btn {
+  width: 140px;
+  height: 48px;
+  font-weight: 500;
+  font-size: 17px;
```
Comment (tugsem) [inline style.css:480]: - [ ] Kindly add button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template:
image

Comment (samuelmoiwa) [reply style.css:480]: Hi, I think the style is not supposed to be applied to the ***Get In Touch*** button. please recheck.

Comment (tugsem) [reply style.css:482]: image

Comment (tugsem) [reply style.css:480]: Hi @samuelmoiwa I think you missed that point in the project requirements. Here's a screenshot: image

**Review by tugsem — APPROVED** [review]
Hi @samuelmoiwa >, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Highlights ⭐ 
✅ Button interaction styles implemented
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (samuelmoiwa) [reply style.css:480]: but if the hover style is added to the button, it blends in with the background colour making the button invisible, don't know, but I think this button style is only applied to the languages and see project button.

**Pull request merged**